internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -115,6 +115,8 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, awsCli
 		}
 	}
 
+	logger = logger.WithField("vpc-id", clusterResources.VpcID)
```

Comment:
@gabrieljackson I felt that having a field that tells me the VPC ID helps when debugging multi-tenant database issues, so I added it here.